_includes/icml/2020/presenters.icml.html

```diff
@@ -0,0 +1,31 @@
+<!-- Speakers Grid Section -->
+<section id="speakers" class="bg-light-gray">
+  <div class="container">
+    <div class="row">
+      <div class="col-lg-12 text-center">
+        <h2 class="section-heading">Speakers</h2>
+        <h3 class="section-subheading text-muted"></h3>
+      </div>
+    </div>
+    <div class="grid">
+      {% assign featuredItems = site.tags.oral-icml2020 | sort: "featuredOrder" %}
+      {% for post in featuredItems %}
+      <div class="cell speakers-item">
+        <a href="{{post.website}}" target="_blank" class="speakers-link">
+          <img src="/assets/img/icml/2020/presenters/{{ post.img }}" class="responsive-image img-centered" alt="" height="300" >
+        </a>
+        <div class="speakers-caption">
+          <h4>{{ post.name }}</h4>
+          <p>{{ post.title }}</p>
+          <h5>{{ post.topic }}</h5>
+
+        </div>
+      </div>
+      {% endfor %}
+    </div>
+  </div>
+</section>
+<section>
+
```

Comment:
The list of poster presenters is missing from this file. See prior workshop pages: https://github.com/latinxinai/latinxinai.github.io/blob/master/_includes/neurips/2019/presenters.neurips.2019.html

This list can be generated programmatically, see how @pablorp80 executed at prior ICML: https://github.com/latinxinai/latinxinai.github.io/blob/master/_includes/icml/2019/make_presenters.py